dapps/digital-twin-detail/src/components/DigitalTwinDetail.ts

```diff
@@ -96,6 +96,9 @@ export default class DigitalTwinDetailComponent extends mixins(EvanComponent) {
           // handle deleted and non-DT contracts
           .catch(() => {
             this.hasError = true;
+            // fill empty twin data, so the sidepanel can be displayed, even in error mode
```

Comment:
IMO putting that kind of logic in catch() block is bit of an anti pattern. We would also need to do this then everywhere, where we use the Twin.  Also setting vars of the instance by reference ... I think it would be better setting those values directly in constructor of DAppTwin as default values.

Reply:
PS: I think it would be totally okay to show only an error page, when the twin can not be initialized.